### PR TITLE
feat(deps): Add Nextcloud 28 support on master

### PR DESCRIPTION
**Pull request by nickvergessen**
Signed-off-by: Joas Schilling <coding@schilljs.com>

### CODE REVIEW

**Review by ChristophWurst — APPROVED** [review]

**Review by ChristophWurst — CHANGES_REQUESTED** [review]

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "suspicious_login",
-  "version": "5.0.0-beta2",
+  "version": "5.1.0-dev",
```
Comment (st3iny) [inline package.json:3]:
```suggestion
  "version": "6.0.0-dev",
```

See above.

File: appinfo/info.xml [hunk 2]
```diff
@@ -7,7 +7,7 @@
 	</summary>
 	<description>Detect and warn about suspicious IPs logging into Nextcloud
 	</description>
-	<version>5.0.0-beta.2</version>
+	<version>5.1.0-dev</version>
```
[the remark this answers is not in the document]
Comment (st3iny) [reply appinfo/info.xml]: Done

File: package.json (hunk 1 above)
Comment (st3iny) [reply package.json]: Done

**Review by st3iny — APPROVED** [review]

**Comment by st3iny** [comment]
CI fix at https://github.com/nextcloud/suspicious_login/pull/802.

**Pull request merged**